### ISSUE TITLE
dns: valid syntax breaks booting of rancherOS

### ISSUE THREAD

**Issue by deniseschannon** [rancherio/os#228]
v0.3.0

Setting DNS in the syntax of the example will cause the reboot to fail at 
```
ip_tables: (C) 2000-2006 Netfilter Core Team
random: nonblocking pool is initialized
```

Example File Syntax:
```yaml
dns: 8.8.8.8
```

**Comment by deniseschannon** [comment]
The actual error while rebooting is:

```
Could not parse config for project system-init: yaml: unmarshal errors:
line 8: cannont unmarshal !! str '8.8.8.8' into []string
```

**Comment by deniseschannon** [comment]
Verified fix v0.3.1-rc1
